File: CS/Inline_Unobtrusive/Models/Customer.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Inline_Unobtrusive.Models {
    public class Customer {
        [Required(ErrorMessage = "This field is required")]
        public int ID { get; set; }

        [StringLength(10, ErrorMessage = "Name cannot contain more than 10 characters")]
        public string Name { get; set; }

        public string City { get; set; }
    }
}

File: VB/Inline_Unobtrusive/Models/Customer.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.Linq
Imports System.Web

Namespace Inline_Unobtrusive.Models
	Public Class Customer
		Private privateID As Integer
		<Required(ErrorMessage := "This field is required")> _
		Public Property ID() As Integer
			Get
				Return privateID
			End Get
			Set(ByVal value As Integer)
				privateID = value
			End Set
		End Property

		Private privateName As String
		<StringLength(10, ErrorMessage := "Name cannot contain more than 10 characters")> _
		Public Property Name() As String
			Get
				Return privateName
			End Get
			Set(ByVal value As String)
				privateName = value
			End Set
		End Property

		Private privateCity As String
		Public Property City() As String
			Get
				Return privateCity
			End Get
			Set(ByVal value As String)
				privateCity = value
			End Set
		End Property
	End Class
End Namespace